lib/api/request/kuzzleRequest.ts

```diff
@@ -848,13 +849,9 @@ export class KuzzleRequest {
     errorName: string,
     def: number | null = null
   ): number {
-    let value = get(obj, name);
+    let value = get(obj, name, def);
 
     if (value === undefined || value === null) {
```

Comment:
a `null` value should not be considered as missing (apply to other part of the code)